lib/cadet_web/controllers/contestvoting_controller.ex

```diff
@@ -0,0 +1,21 @@
+defmodule CadetWeb.ContestVotingController do
+  use CadetWeb, :controller
+  use PhoenixSwagger
+
+  alias Cadet.Assessments
+
+  def show(
```

Comment:
What is this route for?

This seems a bit redundant, as you can just include the vote information as the answer in the assessment.

Reply:
we have fixed this and migrated to assessments